content/docs/addons-test-utils.md

```diff
@@ -234,7 +234,7 @@ findRenderedDOMComponentWithClass(
 )
 ```
 
-Like [`scryRenderedDOMComponentsWithClass()`](#scryrendereddomcomponentswithclass) but expects there to be one result, and returns that one result, or throws exception if there is any other number of matches besides one.
+Seperti [`scryRenderedDOMComponentsWithClass()`](#scryrendereddomcomponentswithclass) tetapi mengharapkan satu hasil, dan mengembalikan satu hasil tersebut, atau melempar protes jika ada lebih dari satu yang cocok.
```

Comment:
*exception* I think doesn't need to be translated to `protes` since it is a common engineering term mas 😄

Reply:
oke, i already changed it ya 😅